src/config.c

```diff
@@ -276,8 +276,9 @@ int load_config(struct vpn_config *cfg, const char *filename)
 			}
 			cfg->otp_delay = otp_delay;
 		} else if (strcmp(key, "cookie") == 0) {
-			free(cfg->cookie);
-			cfg->cookie = strdup(val);
+			log_warn("Ignoring option \"%s\".\n", key);
+		} else if (strcmp(key, "cookie-on-stdin") == 0) {
+			log_warn("Ignoring option \"%s\".\n", key);
```

Comment:
On the other hand, this one might make sense in a config file.

Reply:
I was not sure about it and after thinking about it a little bit I decided to keep this one out.

This option is mostly useful when piping stuff to `openfortivpn`. If you don't write anything to stdin through a pipe the program will wait for you to write something and press enter without prompting for anything. This doesn't seem that intuitive, so I thought that requiring an explicit `--cookie-on-stdin` was a good idea.

I'm also OK with supporting the option in the config file, so I'll change that if you think it is a good idea having it there.

Reply:
I don't know. Let's keep it as is for now.